grbl/spindle_control.h

```diff
@@ -57,10 +57,12 @@ uint8_t spindle_get_state();
 #else
   
```

Comment:
@chamnit This part is the main idea, it fixed a couple build errors then I fixed the other errors which it introduced.